core/shared-preferences/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("appcoins.android.library")
+}
+
+android {
+  namespace = "com.appcoins.wallet.shared_preferences"
+}
+
+dependencies {
+  implementation(libs.androidx.security.crypto)
+  testImplementation("junit:junit:4.13.2")
+  androidTestImplementation("androidx.test.ext:junit:1.1.5")
```

Comment:
we should use junit present in the version catalog

Reply:
Sorry, I forgot to remove it. Done.